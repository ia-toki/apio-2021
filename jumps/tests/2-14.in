209636825ecd2c38299498cd26890ef71dd128c2
200 200
198 197 196 192 191 190 186 185 184 180 179 178 174 173 172 168 167 166 162 161 160 156 155 154 150 149 148 144 143 142 138 137 136 132 131 130 126 125 124 120 119 118 114 113 112 108 107 106 102 101 100 96 95 94 90 89 88 84 83 82 78 77 76 72 71 70 66 65 64 60 59 58 54 53 52 48 47 46 42 41 40 36 35 34 30 29 28 24 23 22 18 17 16 12 11 10 6 5 4 1 2 3 7 8 9 13 14 15 19 20 21 25 26 27 31 32 33 37 38 39 43 44 45 49 50 51 55 56 57 61 62 63 67 68 69 73 74 75 79 80 81 85 86 87 91 92 93 97 98 99 103 104 105 109 110 111 115 116 117 121 122 123 127 128 129 133 134 135 139 140 141 145 146 147 151 152 153 157 158 159 163 164 165 169 170 171 175 176 177 181 182 183 187 188 189 193 194 195 199 200
99 99 199 199
14 116 189 196
6 32 45 187
17 19 196 198
65 101 167 199
110 181 196 196
72 147 157 179
4 54 192 197
0 111 192 197
12 31 131 192
71 134 140 199
63 128 164 192
70 120 181 185
25 56 87 138
26 69 70 138
18 137 180 183
166 178 189 189
9 14 186 191
122 176 183 184
100 181 194 197
11 184 196 197
53 65 166 171
25 182 192 192
1 69 195 198
134 176 184 196
2 33 196 196
0 3 103 170
35 85 157 192
40 151 173 190
5 97 102 150
10 53 77 192
96 120 177 180
15 43 147 193
7 104 170 180
63 156 163 166
110 116 173 196
8 33 136 137
83 97 108 195
5 33 179 185
38 46 132 164
12 69 78 145
18 28 53 89
64 78 155 184
114 190 196 199
92 141 185 195
11 94 186 192
8 82 91 126
60 94 103 195
73 88 151 166
45 79 166 173
16 35 123 137
72 149 170 175
67 83 193 198
22 45 113 161
109 188 197 197
67 104 112 121
3 64 153 166
107 172 184 184
39 101 141 174
46 83 91 102
197 197 198 198
6 24 197 197
4 44 146 190
4 7 98 161
80 98 120 132
38 106 136 143
67 79 119 182
75 188 198 198
11 21 70 183
14 66 130 130
57 66 139 151
7 43 128 181
77 122 123 124
42 130 131 182
4 15 143 157
48 62 138 177
37 133 175 197
5 176 185 198
82 87 96 109
54 77 195 197
26 145 176 190
97 133 156 158
5 90 96 107
87 118 175 178
4 31 195 198
86 177 190 195
69 166 171 189
147 151 157 190
25 67 90 147
145 177 184 198
35 45 165 190
114 171 172 174
48 195 197 199
85 145 148 159
124 181 185 191
21 87 189 192
147 195 199 199
9 171 182 190
34 51 187 189
36 104 130 175
55 186 190 195
66 82 145 147
28 39 155 157
12 33 69 133
12 29 119 180
89 183 184 190
0 2 183 183
84 96 97 187
39 95 145 158
93 144 191 194
0 5 49 143
50 111 133 190
36 165 197 197
3 39 65 171
57 190 192 192
24 133 152 187
24 28 142 154
90 149 161 176
6 8 104 113
56 141 158 171
4 23 153 157
87 108 135 171
28 125 149 150
36 54 85 134
1 2 189 198
59 86 185 187
4 24 83 113
165 182 184 194
78 172 193 199
13 134 138 185
48 142 170 173
2 3 76 94
6 7 199 199
0 0 105 185
10 22 95 150
72 87 177 185
73 92 184 194
6 23 48 87
79 125 175 178
21 130 167 169
94 112 117 164
9 189 191 195
4 154 182 182
70 95 195 195
41 110 199 199
33 194 198 198
1 15 59 136
138 171 196 197
98 164 171 195
74 115 184 193
6 14 77 116
130 138 141 152
8 19 26 137
2 95 151 158
23 159 179 179
20 136 143 196
7 69 185 195
74 139 143 143
1 29 134 166
57 60 157 164
187 193 198 198
151 188 196 198
8 14 135 189
61 142 182 195
13 53 194 197
35 142 152 187
100 180 191 196
87 96 181 187
63 128 163 184
6 63 131 181
57 118 169 176
19 41 142 171
24 31 119 138
0 57 143 160
49 90 169 193
24 126 157 199
43 50 184 194
95 157 179 193
27 193 199 199
21 154 188 189
48 144 146 168
51 51 149 151
40 68 148 173
122 148 161 163
7 9 149 181
42 105 109 110
51 127 153 155
40 85 101 169
13 130 168 192
15 40 144 167
0 63 68 86
129 197 199 199
106 117 179 187
42 51 95 149
1 14 42 43
71 116 126 178
18 127 160 199
82 107 127 135
10 147 187 195
72 84 111 147
