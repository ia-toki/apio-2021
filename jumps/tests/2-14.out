fc5005cd102fffc504bc1615010ebf88823d65f3
OK
67
3
1
10
23
11
7
1
1
1
5
20
36
-1
1
1
8
1
5
10
6
14
7
1
6
2
-1
1
11
1
1
38
1
1
5
39
-1
1
1
1
1
-1
15
5
30
1
1
1
18
9
-1
15
42
1
7
1
1
9
1
-1
1
4
1
-1
1
1
1
7
1
-1
1
1
1
1
-1
1
10
1
-1
34
3
16
-1
39
1
9
4
5
1
5
1
1
2
3
4
8
3
1
16
1
3
9
-1
-1
1
1
-1
1
1
32
-1
1
22
1
2
1
-1
9
-1
11
-1
16
1
-1
1
31
-1
2
15
1
14
-1
5
-1
-1
34
40
-1
34
1
4
2
3
45
28
3
-1
17
5
40
-1
2
-1
1
3
1
1
4
-1
11
4
6
1
28
7
1
8
47
19
1
19
1
-1
1
14
1
20
16
5
8
1
2
1
10
-1
2
4
1
1
1
-1
2
42
1
-1
1
1
8
1
1
